src/main/java/com/example/DoughnutShop.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+public class DoughnutShop {
+
+    public String prepareDoughnut(String doughnutType) {
```

Comment:
doughnutType is a string but perhaps we could introduce an Enumeration type called DoughnutType which contains all the possible types of doughnuts. This may make it easier to extend the functionality of this method when we add more types of doughnuts.